### PR TITLE
[CONTENT] Leader's Den Fails

### PR DESCRIPTION
**Pull request by tobes03**
## About The Pull Request

A few very simple failure outcomes for the leader's den interactions with outsiders. This is basically just so our bases are covered

## Why This Is Good For ClanGen

outcome :)

## Proof of Testing

image
image
image
image

### CODE REVIEW

File: resources/dicts/events/leader_den/fail/outsider.json [hunk 1]
```diff
@@ -365,5 +365,230 @@
...
+  },
+  {
+    "interaction_type": "invite",
+    "event_text": "c_n finds m_c behind some sort of sheet of clear, unmeltable ice, barricading {PRONOUN/m_c/object} inside the Twoleg nest. It's impossible to carry a conversation from outside.",
+    "reputation": ["any"],
+    "rep_change": 0,
+    "m_c": {
+      "status": ["kittypet"],
+      "new_thought": "Curiously watches the Clan cats retreat",
+      "relationships": [
```
Comment (scribblecrumb) [inline resources/dicts/events/leader_den/fail/outsider.json]: I don't think we need to add dislike for this one

File: resources/dicts/events/leader_den/fail/outsider.json [hunk 2]
```diff
@@ -365,5 +365,230 @@
...
+      "status": ["kittypet"],
+      "new_thought": "Shivers in {PRONOUN/m_c/poss} nest at the thought of Clan cats"
+    }
+  },
+  {
+    "interaction_type": "hunt",
+    "event_text": "When c_n finds m_c, {PRONOUN/m_c/subject} {VERB/m_c/greet/greets} the Clan cats politely. m_c is stunned when c_n descends on {PRONOUN/m_c/object} with unsheathed claws, but {PRONOUN/m_c/subject} {VERB/m_c/react/reacts} just in time to sprint off.",
+    "reputation": ["welcoming"],
+    "rep_change": 0,
+    "m_c": {
```
Comment (scribblecrumb) [inline resources/dicts/events/leader_den/fail/outsider.json:550]: lets add a dislike rel change here too

File: resources/dicts/events/leader_den/fail/outsider.json [hunk 3]
```diff
@@ -365,5 +365,230 @@
...
+      "status": ["kittypet", "loner", "rogue"],
+      "new_thought": "Is stunned to discover Clan cats are much more vicious than their reputation suggested"
+    }
+  },
+  {
+    "interaction_type": "hunt",
+    "event_text": "As soon as the c_n cats slink into view, m_c turns tail and sprints off. Every cat in the area knows how c_n treats outsiders.",
+    "reputation": ["hostile"],
+    "rep_change": 0,
+    "m_c": {
```
Comment (scribblecrumb) [inline resources/dicts/events/leader_den/fail/outsider.json:569]: and a dislike change here

**Review by scribblecrumb — CHANGES_REQUESTED** [review]

**Comment by tobes03** [comment]
good calls 🙏

**Review by scribblecrumb — APPROVED** [review]

**Pull request merged**